### DISCUSSION TITLE
Remove/Hide "Videos feed last updated: # minutes ago"

### DISCUSSION THREAD

**Discussion by TheGalaxiaN** [FreeTubeApp/FreeTube#5310] [Q&A]
Been using Freetube for awhile now, thank you devs for the hard work. As of the recent 0.21.0 beta appimage there is a new bar that displays the feed update notification, i however have missed or there is no way to toggle this off.
Is there a setting somewhere where i can hide or remove this bar?

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
> Is there a setting somewhere where i can hide or remove this bar?

There isnt a setting to hide that as this is important information we need to display to the user. This way the user can track easier whats new on their subscription page. Together with https://github.com/FreeTubeApp/FreeTube/issues/4536 will make it even easier for users to see whats new.

**Reply by 4yman-0** [reply]
>There isnt a setting to hide that as this is important information

But still, we should give an option to the users if they don't want the latest videos from 10 minutes or something.

**Reply by TheGalaxiaN** [reply]
Just to clarify, i don't wish to disable the function and agree that a notification as to the last refresh is important, however i do hope for a setting/toggle to remove or hide the bar. 
Can it not be made to display once when the update is done then hide? Personally i find it an ugly addition to a very nice interface, the function is relevant the bar across the interface is not.

**Reply by kommunarr** [reply]
By not disabling the function but "hid[ing] the bar", do you mean it still showing in the regular place but not following you as you scroll down? We're unlikely to implement any particular setting with requests pertaining to hiding/disappearing the bar in the foreseeable future due to our current set of priorities.
